content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
+
+#### Situation
+Used only in the year where the Material Type is Narrative
+Contains two parts to the question:
+- Quotation
+    - Must be a direct quotation from Scripture by a person or an entity. 
+    - The quotation must be reasonable in length, not exceeding two consecutive verses. 
+    - The first five words must be unique to all Quotations.
+    - The quizzer is not required to repeat the quotation word-perfect but must be substantially close without leaving out key components of the quotation.
+    - Must start at the beginning of a quotation or the beginning of a sentence within the quotation.  When a Situation question comes from a quotation that runs for more than 2 verses, the beginning of the Situation question must start within the first 2 verses of the quotation.
+    - If the Quotation is split-up by non-quotation words, the Quotation will be read without the non-quotation words.
+        - A situation question can combine two separate quotations as long as no words are added and the two quotes flow easily from one to another.  For example, Matthew 4:19 says, “Come, follow me”, Jesus said, “and I will send you out to fish for people.”  This example could be written, Who said it?  “Come, follow me, and I will send you out to fish for people” Answer:  Jesus.
+- Questions regarding the Situation of the Quotation
+    - Ask for who said it, to whom, how, why, when, where, or about whom. No other interrogatives are permissible.
+    - Can only ask for a maximum of 3 Questions
+The answer to the Situation Question must be in context.
+
+### Question Structure
+This defines how a Quizmaster must introduce and read each question type.
+
+4 parts: (1) Introduction part 1, (2) Introduction part 2, (3) Final Statement, (4) Actual Question
+(1) "Question Number [ 1, 2, 3, 4, 5, etc ] is a [ TYPE ] question." 
+(2) For Situation questions only: "I need to know [ who said it, how was it said, where was it said, etc ]"
+(3) "Question Number [ 1, 2, 3, 4, 5, etc ] question"
+(4) 
+- For INT and MA, begin reading the question
+- For Chapter Reference (and CRMA), "According to Chapter x, ...."
+- For Chapter Verse Reference (and CVRMA, "According to Chapter x, Verse x, ......"
+- For Quote Questions, "Quote [ Book Name ], Chapter [Chapter Number], Verse [ Verse Number]"
+- For Quote these Two Verses Questions. "Quote [ Book Name ], Chapter [Chapter Number], Verses [ Verse Number 1] and [Verse Number 2]"
+- For all Finish Questions (and sub-types), begin reading the verse(s)
+- For Situation Questions, begin reading at the Quotation.
+
 ### Invalid Questions
+Invalid Questions – Questions should be declared invalid by the quizmaster or answer judge(s) if:
+- The answer requires interpretation by the quizzer.
+- The question is ambiguous, tricky, or misleading.
+- The question might have two different or conflicting answers.
```

Comment:
I really want to take this one out.

Reply:
Why? Because it's redundant?

Reply:
I don't know what it means. Where could a question have "two different answers"???

What are "conflicting answers" ??

I've NEVER applied this ever, and I've written a lot of questions.

Reply:
"Two different or conflicting answers" sounds like every MA question ever. I'm cool with deleting that one line, or moving it to apply to only INT, CR, and CVR.

Reply:
LOL, you're right, please Jeremy or Zach, let me purge this line!!

Reply:
I actually see potential applications of the rule, but it would usually also be covered by other rules as well. An application could be a question like, "God spoke to our forefathers through the prophets when?" with the answer "at many times" but a quizzer says "In the past."

I think the "tricky or misleading" rule would 100% cover these sorts of situations so I would be OK with getting rid of this rule, especially with the way it's inconsistent with the purpose of MA questions.